ui/tests/integration/components/form-field-test.js

```diff
@@ -159,6 +159,27 @@ module('Integration | Component | form field', function (hooks) {
     assert.strictEqual(model.get('foo'), selectedValue);
     assert.ok(spy.calledWith('foo', selectedValue), 'onChange called with correct args');
   });
+  test('it renders: radio buttons for possible values, labels, and subtext', async function (assert) {
```

Comment:
Above there is a conditional `{{or value.label value.value val}}`

could we add another possible value without a label and make sure it renders the expected label? 
```js
{ subText: 'Some subtext 3', value: 'SHA256' },`

Reply:
And renders the subtext*